tests/foreman/api/test_reporttemplates.py

```diff
@@ -459,6 +459,87 @@ def test_positive_applied_errata(
     assert res[0]['issued']
 
 
+@pytest.mark.tier2
+@pytest.mark.rhel_ver_match('[^6]')
+@pytest.mark.no_containers
+def test_positive_applied_errata_by_search(
+    function_org, function_location, function_lce, rhel_contenthost, target_sat
+):
+    """Generate an Applied Errata report
+
+    :id: 0f7d2772-47a4-4215-b555-dd8ee675372f
+
+    :setup: A Host with some applied errata.
+
+    :steps:
+
+        1. Generate an Applied Errata report
+
+    :expectedresults: A report is generated with all applied errata listed
+
+    :CaseImportance: Medium
+    """
+    activation_key = target_sat.api.ActivationKey(
+        environment=function_lce, organization=function_org
+    ).create()
+    cv = target_sat.api.ContentView(organization=function_org).create()
+    ERRATUM_ID = str(settings.repos.yum_6.errata[2])
+    errata_name = (
```

Comment:
Currently causing PRT to fail. It's likely that you'll want to look for errata on satellite, 
after calling the method `setup_org_for_a_custom_repo`, 

otherwise the repository probably does not exist/is not synced yet, and so errata are not present.
Locally it probably works because the errata still exists/synced on your local sat, from a persistent repo existing somewhere.
```suggestion
    target_sat.cli_factory.setup_org_for_a_custom_repo( 
         ... 
    )
    errata_name = ( target_sat.api.Errata().search ( ... ))
```